lib/page/term_table.rb

```diff
@@ -139,7 +139,7 @@ def popolo
     end
 
     def popolo_file
-      @popolo_file ||= EveryPolitician::GithubFile.new(house[:popolo], house[:sha])
+      @popolo_file ||= EveryPolitician::GithubFile.new(house[:popolo], house.sha)
```

Comment:
Do some of the other TODOs mean that we can't just go directly to `house.popolo` here yet?